### PR TITLE
:bricks: (fs): Add /fs directory structure

### PR DESCRIPTION
**Pull request by ladislas**
## Content

- [x] Images
    - [x] Emotions
    - [x] Pictos
    - [x] Battery
    - [x] Logo
- [x] Videos
- [x] Audios
- [x] Web certificates
- [x] Configurations & Informations
- [ ] OS v1.0.0

## Comments

- [x] https://github.com/leka/LekaOS/pull/600#discussion_r861923033
- [x] https://github.com/leka/LekaOS/pull/600#discussion_r861923520
- [x] https://github.com/leka/LekaOS/pull/739#discussion_r861924662
- [x] ~~https://github.com/leka/LekaOS/pull/739#discussion_r861924809~~
- [x] ~~https://github.com/leka/LekaOS/pull/740#discussion_r861925877~~
- [x] https://github.com/leka/LekaOS/pull/796
- [x] https://github.com/leka/LekaOS/pull/806
- [x] https://github.com/leka/LekaOS/pull/816

## Previous PR

- [x] https://github.com/leka/LekaOS/blob/3aa202a48bd617649bfb9b300a99b292c034a08f/app/bootloader/main.cpp#L91
- [x] https://github.com/leka/LekaOS/blob/3aa202a48bd617649bfb9b300a99b292c034a08f/app/bootloader/main.cpp#L157-L160
- [x] https://github.com/leka/LekaOS/blob/64cb58e8481a6744d59e578c76eff24db4bf1106/app/os/main.cpp#L103

## Misc

- [ ] 14+ changes in `libs/BehaviorKit/source/BehaviorKit.cpp` + RobotController UT
- [x] 1 in changes `libs/ConfigKit/include/Config.h` + its UT
- [ ] 4+ in changes `spikes/lk_lcd/main.cpp`

### CODE REVIEW

**Review by YannLocatelli — CHANGES_REQUESTED** [review]
Following files must be updated too:

- [ ] 14+ changes in `libs/BehaviorKit/source/BehaviorKit.cpp` + RobotController UT
- [ ] 1 in changes `libs/ConfigKit/include/Config.h` + its UT
- [ ] 4+ in changes `spikes/lk_lcd/main.cpp`

Note. The one in `app/bootloader/main.cpp` will be modified in incoming related PR #600

---

It is also possible to make these changes in another update, but `FirmwareKit` and `lk_file_reception` spike changes should go with those too to avoid confusion

File: fs/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+LekaOS' filesystem is inspired by the unix [filesystem](https://en.wikipedia.org/wiki/Unix_filesystem).
+
+## Directory layout
+
+`/fs` is the root of the filesystem and directory layout. Everything else is located under `/fs`
+
+|Directory|Description|
+|---|---|
+| `/etc` | "Edit To Configure" - Contains system-wide configuration files, e.g. battery charging hysteresis, number of reboots before factory rollback, screen saver timeout, etc. Tout ce qu'on veut avoir comme configs |
```
Comment (YannLocatelli) [inline fs/README.md:11]: Some french here

File: fs/usr/share/reboots [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (YannLocatelli) [inline fs/usr/share/reboots:1]: Rename `global_reboots_counter`

**Comment by YannLocatelli** [comment]
Rebased - Fix conflict of removed path in lk_lcd in #796

File: fs/etc/bootloader-reboots_limit [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (ladislas) [inline fs/etc/bootloader-reboots_limit:1]: set value

File: app/bootloader/main.cpp [hunk 4]
```diff
@@ -158,8 +158,8 @@ namespace config {
 
 	auto bootloader_version = Config {"/fs/conf/bootloader_version", bootloader::version};
 	auto battery_hysteresis_offset =
-		Config {"/fs/conf/bootloader_battery_hysteresis_offset", battery::default_hysteresis_offset};
-	auto factory_reset_limit = Config {"/fs/conf/factory_reset_limit", factory_reset::default_limit};
+		Config {"/fs/etc/bootloader-battery_hysteresis", battery::default_hysteresis_offset};
+	auto factory_reset_limit = Config {"/fs/etc/bootloader-reboots_limit", factory_reset::default_limit};
```
Comment (ladislas) [inline app/bootloader/main.cpp:162]: cherry pick after ec64b7e2160ccb2fb21e2abf43f5f8bf13db5d04 

update variable names

**Pull request closed without merging**